tests/test_crawlers.py

```diff
@@ -391,6 +406,21 @@ def test_intersects_time_range_no_upper_limit(self):
         self.assertFalse(crawler._intersects_time_range(None, datetime(2019, 2, 19)))
 
 
+class URLNavigationTestCase(unittest.TestCase):
```

Comment:
that can go in the `OpenDAPCrawlerTestCase`, non need for a new class

Reply:
Done in the next commit

Reply:
Done in the next commit